packages/cssnano/src/__tests__/postcss-merge-longhand.js

```diff
@@ -154,3 +154,38 @@ test(
     'h1{margin:10px 20px 30px 40px}',
     'h1{margin:10px 20px 30px 40px}',
 );
+
+test(   
+    'should merge custom props and dont remove fallbacks',
+    processCss,
+    'h1{padding-top:10px;padding-right:15px;padding-bottom:20px;padding-left:25px;padding-top:var(--variable);padding-right:var(--variable);padding-bottom:var(--variable);padding-left:var(--variable)}',
+    'h1{padding:var(--variable);padding:10px 15px 20px 25px}'
```

Comment:
Shouldn't it be: 
```
 h1{padding:10px 15px 20px 25px;padding:var(--variable)}
```
?

Reply:
If we look at many other tests that contain more than one unmegable rule, they have similar behavior. 

Here is some examples from same test file:
```javascript
test(
    'should not merge padding values with mixed !important',
    processCss,
    'h1{padding-top:10px!important;padding-right:20px;padding-bottom:30px!important;padding-left:40px}',
    'h1{padding-bottom:30px!important;padding-left:40px;padding-right:20px;padding-top:10px!important}',
);

test(
    'should not merge identical border values with mixed !important',
    processCss,
    'h1{border-top:1px solid #000;border-bottom:1px solid #000;border-left:1px solid #000!important;border-right:1px solid #000!important}',
    'h1{border-bottom:1px solid #000;border-left:1px solid #000!important;border-right:1px solid #000!important;border-top:1px solid #000}',
);
```
so I guess this is correct behavior

Reply:
I disagree, the semantics have fundamentally changed. See my other comment.

Reply:
@andyjansson done, thanks for help!